nodes/Soroban/actions/swapAssets/swap/helpers.ts

```
@@ -0,0 +1,87 @@
+import { Asset, Operation, Server } from 'soroban-client';
+import axios from 'axios';
+
+export async function getSwapAssetsOperation(
+	server: Server,
+	sourceAsset: Asset,
+	destinationAsset: Asset,
+	amount: string,
+	publicKey: string,
+	slippageAmount: string,
+) {
+	if (!slippageAmount) {
+		return 'Please, select a slippage tolerance';
```

Can we throw an error here?